### ISSUE TITLE
html5fallback.js generates errors

### ISSUE THREAD

**Issue by cr2a-graphique** [joomla/joomla-cms#6573]
Hi, with my fellow worker @vinespie we found a mistake are generate by html5fallback.js

example : 
For the mod_login, when you go out of a field like username "html5fallback.js" generate a aria-invalid="false" in the label.

aria-invalid is not used for label.

**Comment by zero-24** [comment]
Closing as we have a PR: https://github.com/joomla/joomla-cms/pull/6631 Thanks

**Comment by vinespie** [comment]
According to accessibility standards attribute "aria-invalid" should be only placed on the attribute "input" not on the label coresponding.
https://www.w3.org/WAI/GL/wiki/Using_Aria-Invalid_to_Indicate_An_Error_Field

## Actual result
1. Go to "index.php?option=com_users&view=login&Itemid=233" or login page
2. Do not enter a login information and click the button
3. You must see the input login incorrect, the label "Username" with attribute "aria-invalid" set to true.


![2015-05-25 10_39_39-login](https://cloud.githubusercontent.com/assets/5129879/7794392/5567718c-02ca-11e5-94ce-e3e0da752c22.jpg)


## Test instructions
1. Apply the patch
2. Go to "index.php?option=com_users&view=login&Itemid=233" or login page
3. Do not enter a login information and click the button
4. You must see the input login incorrect, the label "Username" with  no attribute "aria-invalid"


![2015-05-25 10_41_18-login](https://cloud.githubusercontent.com/assets/5129879/7794396/66c11c80-02ca-11e5-9d5c-4c4fd37c77a1.jpg)
